src/interface/ttk-thirdparty-api-template.yaml

```diff
@@ -1053,3 +1288,72 @@ components:
         - transactionId
         - transactionRequestId
         - quote
+    DFSPLoginRequest:
+      title: DFSPLoginRequest
+      type: object
+      description: 'POST /login request'
+      properties:
+        userName:
+          $ref: '#/components/schemas/UserName'
+        password:
+          $ref: '#/components/schemas/Password'
+        consentRequestId:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/schemas/CorrelationId.yaml'
+      required:
+        - userName
+        - password
+        - consentRequestId
+    DFSPAuthorizeRequest:
```

Comment:
For web flow, the user needs to provide an updated list of the accounts they wish to grant consent for: https://github.com/mojaloop/pisp/tree/master/docs/linking#141-web

Reply:
OK